compiler/include/llvmUtil.h

```diff
@@ -27,6 +27,7 @@
 #include "llvmVer.h"
 
 #include "llvm/IR/IRBuilder.h"
+#include "llvm/IR/MDBuilder.h"
```

Comment:
I don't think llvmUtil should include MDBuilder. It includes IRBuilder.h only because it uses llvm::IRBuilder<> in the prototypes below.